src/shaderlib/fp64/math-fp64.glsl

```diff
@@ -218,7 +220,7 @@ vec2 sqrt_fp64(vec2 a) {
 }
 
 vec2 exp_fp64(vec2 a) {
-  const float k = 512.0;
+  const float k = 16.0;
```

Comment:
Maybe a comment that indicates the significance of this constant and possible values. And/or a longer, self explanatory variable name.